README.md

```diff
@@ -36,6 +43,15 @@ Please keep the following in mind:
 
 - Each exercise should have a test suite, an example solution, a template file for the real implementation and an `elm-package.json` file with the `elm-test` and `elm-console` dependencies. The CI build expects files to be named using the following convention. The example solution should be named `ExerciseModuleName.example`. The template file should be named `ExerciseModuleName.elm`. Test file should be named `ExerciseModuleNameTest.elm`.
 
+- The recommended workflow when working on an exercise is to first create the implementation and test files, `ExerciseModuleName.elm` and `ExerciseModuleNameTest.elm`.
+  - Test the new exercise directly by running `elm-test exercises/exercise_module_name/ExerciseModuleNameTest.elm`. 
+
+  - Once the implementation of the exercise is complete, move `ExerciseModuleName.elm` to `ExerciseModuleName.example` and create the template file.
+
+  - Make sure everything is good to go by running all tests with `bin/build.sh`.
+
+- Please do not commit any Elm configuration files inside the exercise directories, `elm-package.json` or `elm-stuff`.
```

Comment:
This is actually no longer quite true... we actually *want* an elm-package.json in the exercise folder, but it should be the "standard" one.

Looks great otherwise :+1: